diff --git a/node_modules/@types/leaflet/index.d.ts b/node_modules/@types/leaflet/index.d.ts
index d9133ef..3d9bcdb 100644
--- a/node_modules/@types/leaflet/index.d.ts
+++ b/node_modules/@types/leaflet/index.d.ts
@@ -1793,6 +1793,7 @@ export class Map extends Evented {
     tap?: Handler;
     touchZoom: Handler;
     zoomControl: Control.Zoom;
+    pm: any;
 
     options: MapOptions;
 }
@@ -1903,6 +1904,8 @@ export class Marker<P = any> extends Layer {
 
 export function marker(latlng: LatLngExpression, options?: MarkerOptions): Marker;
 
+export function markerClusterGroup(): LayerGroup;
+
 export namespace Browser {
     // sorting according to https://leafletjs.com/reference-1.5.0.html#browser
     const ie: boolean;
